cmd/frontend/graphqlbackend/repositories.go

```diff
@@ -130,6 +112,12 @@ func (r *repositoryConnectionResolver) compute(ctx context.Context) ([]*types.Re
 			}
 		}
 
+		if !r.cloned {
+			opt2.NoCloned = true
+		} else if !r.notCloned || !r.cloneInProgress {
```

Comment:
I don't have much context around this code, but feel like it should be the following?
```suggestion
		} else if !r.notCloned && !r.cloneInProgress {
```

Reply:
It's indeed very confusing so here's the context: `notCloned` and `cloneInProgress` are true by default, so this condition is valid only if one of them has been explicitly set to false by the client.

Reply:
Ah... I see, please add what you said as comments :)

Reply:
Good idea :)